docker-app/qfieldcloud/core/pagination.py

```diff
@@ -1,15 +1,46 @@
+from typing import Callable
+
+from qfieldcloud.settings import QFIELDCLOUD_API_DEFAULT_PAGE_LIMIT
```

Comment:
```suggestion
from django.conf import settings
```

Imports of the settings should be like this, so Django can do a bit of magic and merge different app settings into one. Otherwise you might miss some overwrite of setting values from another app.